translation/debug.rst

```diff
@@ -162,12 +162,12 @@ translation file in the ``fr`` locale and run the command, you will get:
     ----------  ------------------  ----------------------  -------------------------------
      State      Id                  Message Preview (fr)    Fallback Message Preview (en)
     ----------  ------------------  ----------------------  -------------------------------
-     fallback    Symfony is great    J'aime Symfony          Symfony is great
+     fallback    Symfony is great    Symfony is great         Symfony is great
```

Comment:
same as above

Reply:
Exactly! That's all I've done, this messages must be the same!

Reply:
This part is in conflict with the other merged PR.